spec/requests/06_delete_actually_deletes_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe "DELETE /ratingQuestions/:id" do
+  context "with an existing question" do
+    let!(:question) do
+      RatingQuestion.create!(title: "Hello World!", tag: "World")
+    end
```

Comment:
Put a space here between the `let` and the `it`.

Reply:
ok